KillTheBill/Assets/Scripts/GameStateManager.cs

```diff
@@ -46,6 +63,9 @@ void FixedUpdate()
         if(Input.GetKeyDown(KeyCode.F)){
             _currentGameState = GameState.GS_WIN;
         }
+        if(Input.GetKeyDown(KeyCode.L)){
+            _currentGameState = GameState.GS_LOSE;
+        }
```

Comment:
Right now it's for debugging but don't forget to remove it before releasing the game!

 Or we can add a boolean `_isDebug` to this class and default it to "false", only in our editor that we set it to "true" so that we can enable this behavior.